### PR TITLE
Views/avaliable events

### PR DESCRIPTION
**Pull request by jnaraujo**

### CODE REVIEW

**Review by legotardo — APPROVED** [review]

File: components/eventCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import { Text, View, Image, TouchableHighlight } from "react-native";
+import styles from "./styles";
+import { useNavigation } from "@react-navigation/native";
+
+
+import { arrowOrangeSvg } from "./../../assets/CDN/svg";
```
Comment (CarlosZoft) [inline components/eventCard/index.tsx]: tem o index la viu hahaha

**Review by CarlosZoft — CHANGES_REQUESTED** [review]

File: components/eventCard/index.tsx (hunk 1 above)
Comment (CarlosZoft) [reply components/eventCard/index.tsx]: import { arrowOrangeSvg } from "./../../assets/CDN"

File: components/eventCard/index.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import styles from "./styles";
 import { useNavigation } from "@react-navigation/native";
 
 
-import { arrowOrangeSvg } from "./../../assets/CDN/svg";
+import { arrowOrangeSvg } from "./../../assets/CDN/";
```
Comment (CarlosZoft) [inline components/eventCard/index.tsx:6]: tira essa / ai

**Review by CarlosZoft — CHANGES_REQUESTED** [review]

File: components/eventCard/index.tsx (hunk 2 above)
Comment (CarlosZoft) [reply components/eventCard/index.tsx:6]: import { arrowOrangeSvg } from "./../../assets/CDN"

**Review by CarlosZoft — APPROVED** [review]

**Pull request merged**